NovaCustom-PD50-20231126-144950.yml

```diff
@@ -0,0 +1,60 @@
+layout:
```

Comment:
First line needs to be `---` to start the document. Missing this will break things. Here too, you need to replace all `‘` and `’` with `'`.  Also for some reason the values are indented by only one space instead of two. Please make it two for consistency.